### PR TITLE
Added documentation for change graph feature

### PR DESCRIPTION
**Pull request by boldtrn**
As reported in the forum, we miss a documentation about the change graph feature, so I added a description on how to get started.

I also found a bug, reported in #1132. Maybe we fix this bug before merging this documentation?

### CODE REVIEW

**Comment by boldtrn** [comment]
What is the state of this PR?

**Comment by karussell** [comment]
> I also found a bug, reported in #1132. Maybe we fix this bug before merging this documentation?

I would also vote for this :)

File: docs/core/change-graph.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+Currently, it is possible to change the speed of edges as well as the access value of edges.
+
+### Before you start
+
+You should be aware that changing either the access or the speed will not influence the results of ch requests.
+We recommend using `prepare.ch.weightings=no` or you should know what you are doing. This can be used in combination
+with [Landmarks](landmarks.md), but only if you increase the weight, decreasing the weight might lead to incorrect
+routing results. Per default the `/change` endpoint is disabled for security reasons, you can however enable the 
+endpoint first. Be aware that this endpoint is not secured and everybody can call it and change the graph. 
+You should also be aware that the changes are not persistent, when you restart the server the changes are lost.
```
Comment (karussell) [inline docs/core/change-graph.md]: Except you call graphHopperStorage.flush()

**Comment by boldtrn** [comment]
Sorry this must have slipped through my mail - I updated the PR.

**Comment by karussell** [comment]
Cool, thanks!

**Pull request merged**